changelog.txt

```diff
@@ -1,5 +1,9 @@
 *** Changelog ***
 
+2021.08.22 - version 2.0.4
+
+* Bump tested WP versions - #106
```

Comment:
I don't think we need to update the change log since there are no changes to functionality.